test/integration/globalization.spec.js

```diff
@@ -0,0 +1,145 @@
+        misc.asyncTaskCreator(replaceApiReferences, [registry, config]),
+        misc.asyncTaskCreator(serviceLatestCreate, [registry, config]),
+        misc.asyncTaskCreator(globalization, [registry, config, _mapMarketsToRegions])
+      ], done);
+
+    });
+  });
+
+  it('should copied documents to proper locations', (done) => {
+
+    let stats;
+
+    eachRegTopic.async(registry, config, done, (topicDetails, cb) => {
+
+      const regions = _mapMarketsToRegions(topicDetails.markets);
+
+      if (!regions.length) return cb();
+
+      // should copied docus to proper dirs
+      regions.forEach((region) => {
+        const createDestinationPath = pathCreator.globalizationDestination(config.skeletonOutDestination, topicDetails, region.code);
+
+        // check copied version dir
+        const destinationPath = createDestinationPath(topicDetails.version, false);
+        stats = fs.statSync(destinationPath);
+        expect(stats.isDirectory()).to.equal(true);
+
+        // check copied latest dir
+        const destinationPathLatest = createDestinationPath('latest', false);
+        if (topicDetails.latest){
+          stats = fs.statSync(destinationPathLatest);
+          expect(stats.isDirectory()).to.equal(true);
+        }
+
+        const isInternal = fs.existsSync(topicDetails.genDocuLocationInternal);
+
+        // check copied to internal
+        const destinationPathInternal = createDestinationPath(topicDetails.version, true);
+        if (isInternal){
+          stats = fs.statSync(destinationPathInternal);
+          expect(stats.isDirectory()).to.equal(true);
+        }
+
+        // check copied latest dir to internal
+        const destinationPathLatestInternal = createDestinationPath('latest', true);
+        if (topicDetails.latest && isInternal){
+          stats = fs.statSync(destinationPathLatestInternal);
+          expect(stats.isDirectory()).to.equal(true);
+        }
+      });
+
+      cb();
+    });
+  });
+
+  it('should replace domain in copied files', (done) => {
+
+    eachRegTopic.async(registry, config, done, (topicDetails, cb) => {
+
+      const regions = _mapMarketsToRegions(topicDetails.markets);
+
+      if (!regions.length) return cb();
+
+      const checkReplaceDomain = [];      
+      regions.forEach((region) => {
+        if(!region.domain)
+          return;
+
+        const createDestinationPath = pathCreator.globalizationDestination(config.skeletonOutDestination, topicDetails, region.code);
+        const destinationPath = createDestinationPath(topicDetails.version, false);
+        checkReplaceDomain.push(_checkReplaceDomain(destinationPath));
+      });
+
+      async.series(checkReplaceDomain, cb);
+    });
+  });
+
+  after((done) => {
+    rimraf(`${config.tempLocation}`, done);
+  });
+
+});
+
+function _checkReplaceDomain(destinationPath){
+  return function(cb){
+    gulp.src(`${destinationPath}/**/*`)
+      .pipe(tap((file) => {
```

Comment:
`tap(file, stream) => {`

You should return second argument from tap because it is a stream.
In sync functions maybe it will be ok (not sure) but if there would be any async call this function wouldn't return cb before it is finshed.
So after `expect` and in `if` it should be `return stream;`